voice/transcription_test.go

```diff
@@ -26,3 +26,25 @@ func TestTranscriptionGetContents(t *testing.T) {
 		t.Fatalf("mismatched downloaded contents")
 	}
 }
+
+func TestCreateTranscription(t *testing.T) {
```

Comment:
It does not look like we are testing if the right end point is called with right data 

Please check this : https://github.com/messagebird/go-rest-api/blob/a737548a56ce0bc1c6860aaed0b4e20fbfd29d26/conversation/message_test.go#L10

Reply:
Couldn't integrated `mbtest` into this package similar to `conversations` package, I will try to check for method and path in some other way.